core/iwasm/fast-jit/fe/jit_emit_compare.c

```diff
@@ -120,7 +121,7 @@ jit_compile_op_i64_compare(JitCompContext *cc, IntCond cond)
 static int32
 float_cmp_eq(float f1, float f2)
 {
-    if (isnan(f1) || isnan(f2))
+    if (isnanf(f1) || isnanf(f2))
```

Comment:
seems `isnanf` is not a standard c99 function in` math.h`

https://en.cppreference.com/w/c/numeric/math